docs/admin-manual/setup/configuration.md

```diff
@@ -3,176 +3,191 @@ title: Configuration
 sidebar_position: 20
 ---
 
-The streaming host configuration for LibreTime is shown in the file `/etc/airtime/liquidsoap.cfg` which is automatically generated by the **Streams** page, found on the **System** menu of the LibreTime administration interface. For this reason, you would not normally edit the streaming configuration manually, as any changes are likely to be overwritten by the administration interface.
+To configure LibreTime, you need to edit the `/etc/airtime/airtime.conf` file. This page describe the available options to configure your installation.
 
-## Database and RabbitMQ hosts
+## General
 
-Optionally, you may wish to edit the file `/etc/airtime/airtime.conf` to set the PostgreSQL database host, and the username and password to connect to the database with.
-
-You can also set options for RabbitMQ messaging and the LibreTime server in this file, although you shouldn't normally need to adjust the defaults unless you are running a large LibreTime system distributed across multiple servers. To run the LibreTime server in demo mode, which changes the greeting on the login page and prevents user accounts from being created or modified, set the value of _demo_ to 1.
-
-```ini title="/etc/airtime/airtime.conf"
-[database]
-host = localhost
-port = 5432
-name = libretime
-user = libretime
-password = libretime
-
-[rabbitmq]
-host = 127.0.0.1
-port = 5672
-user = airtime
-password = XXXXXXXXXXXXXXXXXXXX
-vhost = /airtime
+The `general` section configure anything related to the legacy and API services.
 
+```ini
 [general]
-api_key = XXXXXXXXXXXXXXXXXXXXX
-web_server_user = www-data
-airtime_dir = /usr/share/airtime
-base_url = libretime.example.com
+# The internal API authentication key. This field is required.
+; api_key =
+api_key = f985635ac757e05715bc2d2b1f7d7045
```

Comment:
Maybe set this to a clearly fake key? e.g. `0000000000000000000000000` or `0123456789abcdef`